en/docs/develop/tools/integration-tools/persist-tool.md

```
@@ -165,13 +165,113 @@ stream<Employee, error?> employees = sClient->/employees;
+### bal persist push
+
+```bash
+bal persist push --datastore <datastore> --module <module>
+```
+
+| Flag | Required | Default | Description |
+|---|---|---|---|
+| `--datastore` | Yes | — | Target data store: `mysql`, `mssql`, `postgresql`, or `sqlite` |
+| `--module` | Yes | — | Name of the module containing the data model |
+
+## Supported data stores
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)